DXRFixes/DeusEx/Classes/ScriptedPawn.uc

```diff
@@ -248,6 +248,154 @@ function _TakeDamageBase(int Damage, Pawn instigatedBy, Vector hitlocation, Vect
     ReactToInjury(instigatedBy, damageType, hitPos);
 }
 
+// ----------------------------------------------------------------------
+// HandleDamage()
+// ----------------------------------------------------------------------
+
+function EHitLocation HandleDamage(int actualDamage, Vector hitLocation, Vector offset, name damageType)
+{
+    local EHitLocation hitPos;
+    local float        headOffsetZ, headOffsetY, armOffset;
+
+    // calculate our hit extents
+    headOffsetZ = CollisionHeight * 0.7;
+    headOffsetY = CollisionRadius * 0.3;
+    armOffset   = CollisionRadius * 0.35;
+
+    hitPos = HITLOC_None;
+
+    if (actualDamage > 0)
+    {
+        if (offset.z > headOffsetZ)		// head
+        {
+            // narrow the head region
+            if ((Abs(offset.x) < headOffsetY) || (Abs(offset.y) < headOffsetY))
+            {
+                // don't allow headshots with stunning weapons
+                if ((damageType == 'Stunned') || (damageType == 'KnockedOut'))
+                    HealthHead -= actualDamage * 2;// DXRando: 2x damage like torso
+                else if(HasHelmet())// DXRando: helmet good, basically a nerf to the pistol especially for late game
+                    HealthHead -= actualDamage * 6;
+                else
+                    HealthHead -= actualDamage * 8;
+                if (offset.x < 0.0)
+                    hitPos = HITLOC_HeadBack;
+                else
+                    hitPos = HITLOC_HeadFront;
+            }
+            else  // sides of head treated as torso
```

Comment:
should we change this? or give it some in-between damage multiplier like x3 or x4?